packages/block-editor/src/layouts/constrained.js

```diff
@@ -119,12 +121,12 @@ export default {
 								</InputControlPrefixWrapper>
 							}
 						/>
-						<p className="block-editor-hooks__layout-controls-helptext">
+						<p className="block-editor-hooks__layout-constrained-helptext">
 							{ __(
 								'Customize the width for all elements that are assigned to the center or wide columns.'
 							) }
 						</p>
-					</div>
+					</>
 				) }
 				{ allowJustification && (
```

Comment:
I don't think any Core blocks show this Justification control, but I made sure that the spacing looks correct even with the Justification control showing. (It is correct in trunk, and still correct here)

<img src="https://github.com/user-attachments/assets/a66c5403-7e95-4a98-99b0-f9ee307eab73" alt="With Justification control" width="276">